config/config.go

```diff
@@ -0,0 +1,67 @@
+package config
+
+import (
+	"github.com/aliazizii/IMDB-In-Golang/internal/model"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+type Config struct {
+	DB     *Database
+	Admin  *model.User
+	Logger *Logger
+	Secret string `mapstructure:"SECRET"`
+}
+
+type Database struct {
+	Host     string `mapstructure:"HOST"`
+	Port     string `mapstructure:"PORT"`
+	Name     string `mapstructure:"DB_NAME"`
+	User     string `mapstructure:"DB_USER"`
+	Password string `mapstructure:"DB_PASSWORD"`
+}
+
+type Logger struct {
+	Level   logrus.Level
+	Enabled bool
+}
+
+func Read() (*Config, error) {
+	viper.AddConfigPath("./")
+	viper.SetConfigName("app")
+	viper.SetConfigType("env")
+
+	viper.AutomaticEnv()
+
+	err := viper.ReadInConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	db := &Database{
```

Comment:
You can use a `viper` instance and use Unmarshal on it instead of filling the fields individually.